cmd/config_test.go

```diff
@@ -176,7 +177,10 @@ var _ = Describe("Config", func() {
 
 				r := graphql.NewRequest(query)
 				r.Variables["config"] = config
-				r.Variables["pipelineValues"] = pipeline.PrepareForGraphQL(pipeline.LocalPipelineVars(map[string]string{}))
+
+				pipelineValues, err := json.Marshal(pipeline.LocalPipelineVars())
+				Expect(err).ToNot(HaveOccurred())
+				r.Variables["pipelineValuesJson"] = string(pipelineValues)
```

Comment:
I can't tell how much test coverage we already have for pipeline parameters, but it would be awesome if we had tests for
* the `--pipeline-parameters` flag being recognized and propagated through
* specifically, passing a mix of string and boolean values and confirming those types are preserved in the graphql query